content/blog/2025-08-28-enabling-crb-by-default-for-almalinux10.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Enabling CRB by default for AlmaLinux 10"
+type: blog
+author:
+ name: "Neal Gompa"
+ bio: "Principal Consultant, Velocity Limitless; AlmaLinux Engineering Steering Committee member"
+ image: /users/nealgompa.png
+date: '2025-08-28'
+images:
+  - /blog-images/2025/crb_enabled_almalinux10.png
+post:
+    title: "Enabling CRB by default for AlmaLinux 10"
+    image: /blog-images/2025/crb_enabled_almalinux10.png
+
+---
+
+AlmaLinux OS 10 and AlmaLinux OS Kitten 10 will have the CRB repository enabled by default.
+
+As part of our efforts to continue improving the experience for AlmaLinux users, we are enabling the CRB repository by default.
+This reduces the friction in using software from Fedora Extra Packages from Enterprise Linux (EPEL).
+
+## What is the CRB repository?
+
+The CRB repository is an extra collection of packages that have not been historically made available by default.
```

Comment:
Add a link to some kind of documentation about CRB for anyone unfamiliar with it.